### ISSUE TITLE
tsmuxer has issues with seamless branching with changes in audio setup

### ISSUE THREAD

**Issue by sjpotter** [justdan96/tsMuxer#633]
I have "many" blurays that have seamless branching where there's an "initial" distributor m2ts file as part of the playlist that has no audio and then a main m2ts that has audio.  In these cases tsmuxer (along with mkvtoolnix, amongst others.  makemkv is a bit smarter an just throws away the initial m2ts file) chokes on these remuxes into the .ts without any audio at all.  

I'd think it be better to be able to fill in gap with "silent" audio.  This should be relatively straight forward if the rest of the audio is PCM, but might be a bit for compressed formats, or at least would have to understand them and have a sample table to draw from.  But it be nice if it a) detected the situation and notified the user and b) handled what it could handle.

**Comment by jcdr428** [comment]
@sjpotter could you please five examples of movies (and distributors/country release) where we can see this case ?

**Comment by sjpotter** [comment]
I believe I've seen this issue on every single recent kino lober release, will verify later as their logo is silent/no audio (one m2ts) and the main movie is not (another m2ts).  Kino Lober films aren't always pcm, so not quite the simple fix I proposed.

**Comment by jcdr428** [comment]
@sjpotter the solution could be to include a stream delay corresponding to the length of the first track.

**Comment by sjpotter** [comment]
yes or just exclude the logo (what makemkv effectively does), but becomes more complicated with the other bug (mpls in/out marks) that I just posted.  

But as I said, it be nice if the situation was detected and warned about in general, and as I said, tsmuxer (right now) creates a video only ts output in this case.
